### ISSUE TITLE
How to get the current training epoch in a custom loss function, as this parameter is needed in the loss calculation

### ISSUE THREAD

**Issue by GeorgePearse** [BinItAI/visdet#470]
**Original issue:** https://github.com/open-mmlab/mmdetection/issues/11512
**Created:** 2024-03-04
**Last updated:** 2024-03-04

---

*No description provided.*
